report_thesis/src/references.bib

```diff
@@ -596,6 +596,132 @@ @misc{duan_ngboost_2020
+author = {Lu-Ning Li and Xiang-Feng Liu and Wei-Ming Xu and Jian-Yu Wang and Rong Shu},
+keywords = {Laser-induced breakdown spectroscopy, Convolutional neural network, Deep learning, MarSCoDe, Multi-component quantitative analysis},
+abstract = {One major technical difficulty of laser-induced breakdown spectroscopy (LIBS) lies in achieving ideal accuracy of quantitative determination of the multiple chemical components in a target sample. In this study, we propose a LIBS multi-component quantitative analytical method based on the construction of a deep convolutional neural network (CNN). More than 1400 LIBS spectra, collected from 23 China national standard reference materials, were utilized to train the CNN and validate its predictive ability as well. The experiment was implemented by the LIBS system in MarSCoDe, which would be the Mars Surface Composition Detector on the rover of China's first Mars exploration mission in 2020. To evaluate the performance of the CNN, we inspect the root mean square error (RMSE) value of the prediction, with both overall RMSE and component-wise RMSE considered, and we further look into the prediction relative error of each component. We compare the performance of the CNN with that of two alternative schemes based on back-propagation neural network (BPNN) and partial least squares (PLS) regression respectively, with the PLS scheme actually containing two methods, i. e. PLS1 and PLS2. Besides the examination of the specific values of RMSE and relative error, we have also carried out some statistical analysis to endow the comparison with statistical significance. Moreover, we investigate the effect of baseline removal preprocessing upon the predictive ability of each method. The results show that the CNN method has the best performance among the four methods in terms of overall accuracy, no matter the test is based on the spectra with or without baseline removal, and the superiority of the CNN over the other three methods is more significant in the latter case. Since the number of samples is relatively small, the results demonstrated in this work are preliminary and unsuitable for immediate generalization, but they indicate that the CNN-based methodology is a promising tool for LIBS quantitative analysis with good accuracy and high efficiency.}
+}
+
+@article{ngboost_landslide,
+  author = {Kavzoglu, T. and Teke, A.},
+  title = {Predictive Performances of Ensemble Machine Learning Algorithms in Landslide Susceptibility Mapping Using Random Forest, Extreme Gradient Boosting (XGBoost) and Natural Gradient Boosting (NGBoost)},
+  journal = {Arabian Journal for Science and Engineering},
+  volume = {47},
+  pages = {7367--7385},
+  year = {2022},
+  doi = {10.1007/s13369-022-06560-8},
+  url = {https://doi.org/10.1007/s13369-022-06560-8},
+  received = {25 July 2021},
+  accepted = {29 December 2021},
+  published = {17 January 2022},
+  issue_date = {June 2022}
 @book{hastie_elements,
```

Comment:
I guess you were a little... Hastie with resolving merge conflicts ;)

```suggestion
  issue_date = {June 2022},
 }
 
@book{hastie_elements,
```

Reply:
I will hastiely resolve this issue..